TODO

```diff
@@ -0,0 +1,59 @@
+Unprioritized TODOs
+
+OpenID Connect / OAuth2
+
+- Let clients require signing algorithms (see id_token_signed_response_alg)
+- Support ECDSA keys
+- Support client_secret_jwt client authentication
+- Add a "NextSigningKey" to the storage.Keys type so clients can cache more aggressively
+- Support grant_type=password
```

Comment:
suggestion: use the GH checkbox syntax and check these off as they are done. So we can still see the old things after they've been completed.

i.e.

- [ ] an incomplete task
- [x] a complete tast